Doc/glossary.rst

```diff
@@ -754,7 +754,7 @@ Glossary
 
    loader
       An object that loads a module. It must define a method named
-      :meth:`load_module`. A loader is typically returned by a
+      :meth:`!load_module`. A loader is typically returned by a
```

Comment:
It does not say what is the signature and the semantic of that method.

It could be a reference to `importlib.abc.Loader.load_module`.